extensions/wikia/SpecialContact2/SpecialContact.body.php

```diff
@@ -299,13 +303,14 @@ function processCreation() {
 
 		$body = "\n{$this->mProblemDesc}\n\n----\n" . $m_shared . $info;
 
-		if($this->mCCme) {
+		if ( $this->mCCme ) {
 			$mcc = $this->msg( 'specialcontact-ccheader' )->text() . "\n\n";
 			$mcc .= $m_shared . "\n{$this->mProblemDesc}\n";
 		}
 
 		$mail_user = new MailAddress($this->mEmail);
-		$mail_community = new MailAddress( $wgSpecialContactEmail, 'Wikia Support');
+		$mail_support_service = new MailAddress( $wgSpecialContactEmail, 'Wikia Support');
+		$mail_support_from = new MailAddress( self::WIKIA_SUPPORT_EMAIL, 'Wikia Support' );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)